### ISSUE TITLE
[infra] AERA-v26.10 #715 L4 authorization-heading guard repair1

### ISSUE THREAD

**Issue by vinceackermann2-sys** [vinceackermann2-sys/tam-research#716]
## Purpose

Continue the exact frozen v26.10 memory-safe optimization microbenchmark after consumed trigger #715 failed in the GitHub workflow authorization-heading guard before any Modal/GPU execution.

Current authoritative source at issue creation:
- `SOURCE_MAIN=8371d9ab3988df33c7b92b529016618ebf60f9ee`
- `SOURCE_TREE=fd171423b5878c9082e689d7d83da9f8c7c81569`

Consumed predecessor:
- trigger issue `#715`
- workflow run `34145700849`
- job `101817171862`
- attempt `1`
- classification comment `5573729021`
- exact failure: frozen #710 L4 workflow guard searched for authorization heading `## #710 sole L4 memory-safe harness authorization`, while authorization comment `5573681590` began `## #710 sole L4 memory-safe optimization microbenchmark authorization`; `auth_count == 1` failed.

#715 failed before immediate pre-Modal recheck, Modal authentication, remote preflight, GPU allocation, model construction, correctness evaluation, warmups, timing, profiling, result write, or optimization evidence. Never rerun/retry/redispatch #715 or reuse the consumed #710 L4 prefix.

## Frozen scientific/evaluation boundary

Preserve byte-for-byte:
- v26.10 implementation blob `d8f691c198eed1fa96bcbb78a4e76cad82d18779`
- #710 memory-safe evaluator semantics: batch chunk `1`, exact metadata, FP32 corresponding-slice comparison, global max_abs, allclose `atol=rtol=1e-2`
- seed8471 checkpoint bytes/hashes
- batches `(8,64)`
- token seed rule `138471 + 10000 + batch_size`
- hard-sparse routing, `hard=True`, `update_memory=True`, CUDA BF16 autocast
- exact route/expert/depth/active-row/state/schema/checkpoint/sparsity gates
- 3 warmups, 20 unprofiled CUDA-event timing calls per condition, one post-timing profiler call
- PASS requires `>=5%` median latency improvement AND `>=20` fewer `cudaStreamSynchronize` calls at both batch 8 and 64.

Repair orchestration only. Use fresh successor-specific result and trigger namespaces. The successor workflow must bind an exact authorization heading that it itself preregisters; do not depend on the consumed #710 L4 authorization comment as the sole guard.

## Governance

Before any branch/ref move:
1. create successor-specific launcher/wrapper, workflow and CPU/static contract-test blobs as detached Git objects;
2. freeze exact source SHA/tree, #715 failure lineage, all inherited dependency blobs, fresh result path/prefixes, exact authorization heading, new blobs, detached tree and commit in one comment;
3. only then create the named branch;
4. open one immutable PR and consume exactly one original full-repository CPU CI attempt; never rerun/retry/redispatch.

After CPU-green merge and fresh exact-main audit: one fresh zero-GPU preauthorization trigger, then one separate exact-main L4 authorization using the newly frozen heading, then exactly one fresh L4 trigger.

A microbenchmark PASS authorizes only a separately preregistered fresh full frozen Transformer-relative E2E systems gate; it does not itself earn systems PASS. No architecture freeze, S2, fresh scientific seed, independent replication credit, 100M scaling, or breakthrough claim is authorized.

**Comment by vinceackermann2-sys** [comment]
## #716 pre-implementation L4 authorization-heading guard repair1 freeze

This freezes the orchestration-only successor before any #716 branch/ref move. It repairs only the consumed #715 authorization-heading guard mismatch. It creates no Modal/GPU execution, optimization result, full E2E systems run, systems PASS, architecture freeze, S2, fresh scientific seed, replication, scaling, or breakthrough authority.

SOURCE_MAIN=8371d9ab3988df33c7b92b529016618ebf60f9ee
SOURCE_TREE=fd171423b5878c9082e689d7d83da9f8c7c81569
BRANCH=infra/aera-v26-10-issue716-l4-auth-guard-repair1
PREAUTH_PREFIX=[aera-v26-10-issue716-l4-auth-guard-repair1-preauth]
L4_PREFIX=[aera-v26-10-issue716-l4-auth-guard-repair1-l4]
RESULT_PATH=/vol/aera-v26/issue716-v26-10-l4-auth-guard-repair1/result.json
AUTHORIZATION_HEADING=## #716 sole L4 authorization-heading guard repair1 authorization

CONSUMED_TRIGGER=715
CONSUMED_RUN=34145700849
CONSUMED_JOB=101817171862
CONSUMED_ATTEMPT=1
CONSUMED_FAILURE_COMMENT=5573729021
PREVIOUS_AUTH_COMMENT=5573681590

CHECKPOINT_AERA_SHA256=f8aa92421801e8f190247e420632be5f0c20bc5ea8bf6bdeefe06686b3a31b30
CHECKPOINT_TRANSFORMER_SHA256=cdd5cab4439a709468d6607d45d82081b33e876b2e40d91d4a38ba139b219dd7
ISSUE710_LAUNCHER_BLOB=b885250753ea169cd89dd7a978bb3647fd261fe8
ISSUE710_WORKFLOW_BLOB=53e1e22c0b421c51ef6299e777132d7a21dc771d
ISSUE710_CPU_TEST_BLOB=5ce587e83b388a1bda3494d1eff54f4535b483dc
ISSUE706_LAUNCHER_BLOB=e23072aa83b0bc20ccd72e7567f234e9f4a931f6
V26_10_IMPL_BLOB=d8f691c198eed1fa96bcbb78a4e76cad82d18779
ISSUE665_LAUNCHER_BLOB=72f27391ff2f0a7bff8d4532f307ddc4869cf494

LOGIT_COMPARE_BATCH_CHUNK=1
BATCHES=8,64
TOKEN_SEED_RULE=138471+10000+batch_size
ROUTE_MODE=hard_sparse
HARD=True
UPDATE_MEMORY=True
CUDA_AUTOCAST=BF16
WARMUP_CALLS=3
TIMED_CALLS_PER_CONDITION=20
PROFILE_CALLS_PER_CONDITION=1
MIN_LATENCY_IMPROVEMENT=0.05
MIN_STREAM_SYNCHRONIZE_REDUCTION=20
INTEGRATED_ATOL=0.01
INTEGRATED_RTOL=0.01

LAUNCHER_BLOB=84cd58e1f3eed3a51efaff2d7cce2e674c29405f
WORKFLOW_BLOB=4dbdb80245b6767fc07931c45190663d8c9197af
CPU_TEST_BLOB=13f72d319bf685e33978e15b6d88344bc5e7d565
FROZEN_TREE=a208565bdad203e2cfb2354f53c07d022b977d85
FROZEN_COMMIT=c406b4f964c2204dbac1a86b51702109e66a42ba

Exact diff from SOURCE_MAIN: three additive #716 orchestration/contract files only, 657 additions, 0 deletions. Frozen #710 memory-safe launcher and frozen v26.10 implementation are unchanged.

Execution mechanism is frozen as follows: the #716 wrapper invokes the exact merged #710 `preflight` and `run_microbenchmark` bodies through Modal `Function.local()` inside the successor remote container. It temporarily patches only successor identity/result/provenance globals, restores every patched global in `finally`, and contains no copied logit comparator, model execution, CUDA timing, profiler, route/expert/depth logic, optimizer, backward, or training code. The #710 root launcher is explicitly packaged into the Modal image.

The exact future authorization comment MUST begin `## #716 sole L4 authorization-heading guard repair1 authorization`; both the initial L4 workflow guard and the immediate pre-Modal recheck use that identical frozen heading and require `Authorize main: <exact bound main>`.

Superseded detached objects `e338c42e56f6e37565a6f1b2ed63b8868787e6af`, tree `b7f435af279a82dea7b142bea2b27ee13e309de0`, commit `43016f2fe80934cf6e4bb9e3e2ea73e0cca59e17`, plus inert issue #717, receive zero authority and must never be referenced by the #716 branch/PR/triggers/results.

Only after this freeze may the named branch be created at exactly FROZEN_COMMIT, followed by one immutable PR and exactly one original full-repository CPU CI attempt. Never rerun/retry/redispatch a consumed attempt.
